.github/workflows/lint-ui.yaml

```diff
@@ -0,0 +1,32 @@
+name: Lint UI
+run-name: Lint UI - ${{ github.run_id }} - @${{ github.actor }}
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    paths:
+      - 'web/**'
+
+jobs:
+  lint:
+    name: Prettier, ESLint, & TSC
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout OSS Teleport
+        uses: actions/checkout@v3
+
+      - name: Setup Node Toolchain
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+
+      - name: Install Yarn dependencies
+        run: yarn
```

Comment:
`--frozen-lockfile`?

Reply:
> `--frozen-lockfile`?

It doesn't make a difference when using workspaces with yarn v1, yarn v2 fixes this.

https://github.com/gravitational/webapps/blob/0fb98f4606f4d8b1854950f5b923afc7e42a662b/packages/build/scripts/yarn-install-frozen-lockfile.sh